pkgs/development/python-modules/python-easyconfig/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyyaml
+, setuptools
+, six
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-easyconfig";
+  version = "0.1.7";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "RussellLuo";
+    repo = "easyconfig";
+    rev = "v${version}";
+    hash = "sha256-B5Wubw/t6kltwLQdeOrO6QYg4MzIk1MjFXGVMSMp+mA=";
+  };
+
+  # Fix tests, otherwise we get the following error:
+  # Failed: [pytest] section in setup.cfg files is no longer supported, change to [tool:pytest] instead.
+  postPatch = ''
+    substituteInPlace setup.cfg --replace '[pytest]' '[tool:pytest]'
+  '';
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    pyyaml
+    six
+  ];
+
+  pythonImportsCheck = [ "easyconfig" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # TypeError: load() missing 1 required positional argument: 'Loader'
+    "test_valid_yaml_mapping"
+    "test_valid_yaml_mapping_from_envvar"
+  ];
```

Comment:
This is due to a lack of compat with recent pyyaml versions. The library needs to be updated, likely to use `safe_load`. Please send the change upstream.

https://github.com/RussellLuo/easyconfig/blob/master/easyconfig/mappings.py#L40
https://pyyaml.org/wiki/PyYAMLDocumentation#loading-yaml

Reply:
Done.